scripts/lsstinstall

```diff
@@ -292,8 +296,10 @@ elif [ "$exact" = true ] || [ -n "$env_hash" ]; then
         fi
         env_base="https://raw.githubusercontent.com/lsst/scipipe_conda_env"
         url="${env_base}/${env_hash}/etc/conda-${env_platform}.lock"
-    else
+    elif [ -n "$binary_root" ]; then
         url="$binary_root/env/${eups_tag}.env"
+    else
```

Comment:
Because exact envs are only recorded for tarball builds.  An exact env has platform-specific conda packages in it, so it's not meaningful to preserve it with source tags.

Reply:
Ah I would have figured it would have been the resolved env at the time of tagging a given product, but I understand the limitations as they exist currently.

Reply:
Even `conda env export` has platform-specific packages (e.g. `_sysroot_linux-64_curr_repodata_hack=3=h5bd9786_13`).  I don't think there's any way to get a platform-independent "resolved" env beyond the rubin-env version number.